packages/mobile/e2e/src/usecases/WalletConnectV2.js

```diff
@@ -0,0 +1,71 @@
+import WalletConnectClient, { CLIENT_EVENTS } from '@walletconnect/client'
```

Comment:
this spec will never be run because it's skipped in the WalletConnect.spec...i'm not clear on what the plan is for wallect connect v2, it just feels kind of odd to be creating new files that aren't being run, are there plans to enable WC v2 soon?

Reply:
This is just some pre-work for WalletConnectV2 when we turn it on in Valora.